tools/objc/libtool.sh

```diff
@@ -32,15 +32,22 @@ if [ -z ${MY_LOCATION+x} ]; then
   fi
 fi
 
-WRAPPER="${MY_LOCATION}/xcrunwrapper.sh"
+function libtool() {
+  # Just invoke libtool via xcrunwrapper
+  "${MY_LOCATION}/xcrunwrapper.sh" libtool "$@" \
+  2> >(grep -v "the table of contents is empty (no object file members in the"`
+              `" library define global symbols)$" >&2)
```

Comment:
does this grep not matching cause an unexpected failure?

Reply:
I think you could also be less strict here if you wanted

Reply:
When running, I'm seeing things succeed fine when the grep doesn't match. Does that resolve the concern?

Reply:
(I kinda like the full thing to self-document the error, but can remove if you want.)

Reply:
Yes that resolves the concern, I was worried grep would still error. it looks like it's because of the `-v` that it doesn't?

Reply:
Yep, I think so. I was concerned about that, too, when writing